conference/react-js/.eslintrc.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
         "eol-last": ["error", "always"],
         "max-len": [1, 80, 2],
         "react/no-deprecated": "error",
+        "react/prop-types": 0
```

Comment:
1. Remove blank lines, please.
2. Prop types validation is important for preventing bugs, so please enable this rule and define prop types for your components.